### DISCUSSION TITLE
[FF7] Is there any easy way to use cheats?

### DISCUSSION THREAD

**Discussion by malvarenga123** [julianxhokaxhiu/FFNx#669] [Q&A] [answered]
Not counting the shortcuts.
Just like you can in the PSX version using Gameshark/AR codes?

**Comment by julianxhokaxhiu** [comment, marked as answer]
Not at the moment, but there are trainers you can find like Ochu which fits perfectly your case. We do provide "basic cheats" you can find documented in the shortcuts, but we don't plan on adding game hacks as cheats. Hope this helps, cheers!

**Reply by malvarenga123** [reply]
Thanks! I didn't know about Ochu.
